smart-contracts/hardhat.config.js

```diff
@@ -13,13 +13,25 @@ const {
   RINKEBY_API_URL,
   MAINNET_API_URL,
   POLYGON_API_URL,
-  PRIVATE_KEY,
   ETHERSCAN_API_KEY,
   POLYGONSCAN_API_KEY,
   CONTRACT_ADDRESS,
   ROYALTY_RECEIVER_ADDR,
 } = process.env;
 
+// THIS IS A PUBLICLY KNOWN (HARDHAT) PRIVATE KEY. 
+// DO NOT USE THIS IN PRODUCTION OR SEND ANY FUNDS TO THE ASSOCIATED ADDRESS
+const BACKUP_PRIVATE_KEY = "f2f48ee19680706196e2e339e5da3491186e0c4c5030670656b0e0164837257d";
```

Comment:
like this for convenience 👍

Also makes me have an idea, what if we scan public Github projects for people's accidentally published private keys and secret phrases , and send them a message or add an issue that they should burn that wallet